projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/i18n.module.migration.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+  I18N_MODULE,
+  INIT_I18N_CONFIG,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const I18N_MODULE_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: I18N_MODULE,
```

Comment:
we only have `export { I18nModule } from './i18n.module';`. So, function `initI18nConfig` is not exported, right?